lib/screens/customer_product_detail_page.dart

```diff
@@ -126,11 +126,16 @@ class _CustomerProductDetailPageState extends State<CustomerProductDetailPage> {
   Widget _buildStockColorPickerButton(Stock stock, TextStyle textStyle) {
     var _availableStockColors = _getAvailableStockColors(stock);
 
-    _selectedItemColor ??= _availableStockColors.first;
```

Comment:
if `_selectedItemColor` is not used anywhere, please remove the global declaration as well. (must be at the top of this class)